ArduPlane/px4_mixer.cpp

```diff
@@ -42,6 +43,165 @@ bool Plane::print_buffer(char *&buf, uint16_t &buf_size, const char *fmt, ...)
     return true;
 }
 
+/* 
+   create a mixer for a normal angle channel
+*/
+bool Plane::mix_one_channel(char *&buf, uint16_t &buf_size, uint8_t out_chan, uint8_t in_chan)
+{
+    const float limit = 10000;
+    const RC_Channel *inch = RC_Channels::rc_channel(in_chan);
+    const SRV_Channel *outch = SRV_Channels::srv_channel(out_chan);
+    const float scale_out = limit*(outch->get_output_max() - outch->get_output_min()) / (PX4_LIM_RC_MAX - PX4_LIM_RC_MIN);
+
+    bool is_throttle = in_chan==rcmap.throttle()-1;
+    int16_t outch_trim = is_throttle?1500:outch->get_trim();
```

Comment:
Why is throttle trimmed at 1500? Shouldn't it be the halfway point between min/max? (IE on my system 1500 isn't the 50% mark).